guides/v2.3/magento-functional-testing-framework/2.3/configuration.md

```diff
@@ -3,44 +3,68 @@ group: mftf-2.3
 title: Configuration
 functional_areas:
  - Testing
-mftf-release: 2.3.0
+mftf-release: 2.3.6
 redirect_from: /guides/v2.3/magento-functional-testing-framework/2.2/configuration.html
 ---
 
 _This topic was updated due to the {{page.mftf-release}} MFTF release._
 {: style="text-align: right"}
 
-The _*.env_ file provides additional configuration of the Magento Functional Testing Framework (MFTF).
-All MFTF users will need to set the basic configuration values.
-Advanced users may need to set additional values to work with their environment.
+The `*.env` file provides additional configuration of the Magento Functional Testing Framework (MFTF).
+To run the MFTF on your Magento testing instance, specify the basic configuration values.
+For advanced usage, customize the configuration according to your requirements and environment.
 
 ## Basic Configuration
 
 These basic configuration values are required and must be set by the user before MFTF can function correctly.
 
 ### MAGENTO_BASE_URL
 
-* Use: Required.
-* Description: The root URL of the Magento application under test.
-* Example: `http://magento2.vagrant251`
+The root URL of the Magento application under test.
+
+Use: required.
```

Comment:
Removed the _Use: ..._ from the topic.
It is redundant, because at the beginning of the Basic configuration section we mentioned that _basic configuration values are required and must be set by the user_